src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  margin-left: 16px;
+  box-sizing: border-box;
```

Comment:
You've already set box-sizing border-box for all elements. No need to specify it here